src/lib.rs

```diff
@@ -172,6 +174,47 @@ pub trait StreamTools: Stream {
     {
         RecordDelay::new(self)
     }
+
+    /// Merge two streams into one, allowing a custom round robin policy
+    ///
+    /// The resulting stream emits `nb_self` elements from the first stream,
+    /// then `nb_other` from the other. When one of the stream finishes, the
+    /// second is then used.
```

Comment:
Could you also add a comment explaining that `nb_self` and `nb_other` must be non-zero or the method will panic?